q2_moshpit/plugin_setup.py

```diff
@@ -376,52 +384,170 @@
-                      'Diamond database',
+        "input_sequences": "Sequence data of the contigs we want to "
+        "search for hits using the Diamond Database",
+        "diamond_db": "The filepath to an artifact containing the"
+        "Diamond database",
     },
     parameter_descriptions={
-        'num_cpus': 'Number of CPUs to utilize. \'0\' will '
-                    'use all available.',
-        'db_in_memory': 'Read database into memory. The '
-                        'database can be very large, so this '
-                        'option should only be used on clusters or other '
-                        'machines with enough memory.',
-    },
-    outputs=[('eggnog_hits', SampleData[BLAST6]),
-             ('table', FeatureTable[Frequency])
-             ],
-    name='Run eggNOG search using diamond aligner',
+        "num_cpus": "Number of CPUs to utilize. '0' will "
+        "use all available.",
+        "db_in_memory": "Read database into memory. The "
+        "database can be very large, so this "
+        "option should only be used on clusters or other "
+        "machines with enough memory.",
+    },
+    outputs=[
+        ("eggnog_hits", SampleData[BLAST6]),
+        ("table", FeatureTable[Frequency])
+    ],
+    name="Run eggNOG search using diamond aligner",
     description="This method performs the steps by which we find our "
-                "possible target sequences to annotate using the diamond "
-                "search functionality from the eggnog `emapper.py` script",
+    "possible target sequences to annotate using the diamond "
+    "search functionality from the eggnog `emapper.py` script",
 )
 
 plugin.methods.register_function(
     function=q2_moshpit.eggnog.eggnog_annotate,
     inputs={
-        'eggnog_hits': SampleData[BLAST6],
-        'eggnog_db': ReferenceDB[Eggnog],
+        "eggnog_hits": SampleData[BLAST6],
+        "eggnog_db": ReferenceDB[Eggnog],
     },
     parameters={
-        'db_in_memory': Bool,
+        "db_in_memory": Bool,
     },
     parameter_descriptions={
-        'db_in_memory': 'Read eggnog database into memory. The '
-                        'eggnog database is very large(>44GB), so this '
-                        'option should only be used on clusters or other '
-                        'machines with enough memory.',
+        "db_in_memory": "Read eggnog database into memory. The "
+        "eggnog database is very large(>44GB), so this "
+        "option should only be used on clusters or other "
+        "machines with enough memory.",
     },
-    outputs=[('ortholog_annotations', FeatureData[NOG])],
-    name='Annotate orthologs against eggNOG database',
+    outputs=[("ortholog_annotations", FeatureData[NOG])],
+    name="Annotate orthologs against eggNOG database",
     description="Apply eggnog mapper to annotate seed orthologs.",
 )
+
+busco_params = {
+    "mode": Str % Choices(["genome", "transcriptome", "proteins"]),
```

Comment:
should I write a comment about this on the parameter description? Since the BUSCO documentation actually includes these options in the parameter description.

Reply:
Yeah, I would add a comment there. The problem is that if you leave those here, the action will most likely fail so please only keep the `genome`.

Reply:
I added the comment in the parameter description. Is that ok or should I also add a comment on the code?